sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/WhereQuery.cs

```diff
@@ -24,15 +24,30 @@ internal WhereQuery(AdtQueryBuilder parent)
         }
 
         /// <summary>
-        /// Adds a WHERE and its conditional argument(s) clause to the query object. Meant to be used for simple
-        /// conditions involving operators or with basic ADT functions. Multiple WHERE clauses are appended using
-        /// the AND logical operator.
+        /// Adds a WHERE and the conditional arguments for a comparison to the query object. Used to compare ADT properties
```

Comment:
I don't think you need to mention that it adds a `WHERE` to the statement. What if I use it multiple times, does it add the `WHERE` every time?

Reply:
same everywhere.